modules/cloudwatch/dashboard.tf

```diff
@@ -0,0 +1,4 @@
+resource "aws_cloudwatch_dashboard" "applications" {
+  dashboard_name = var.dashboard_name
+  dashboard_body = file("${path.module}/${var.body_file_name}")
```

Comment:
@viktoryathegreat this means file will be picked up from module folder which module users will not be able to do. Now even if I try to pass a file name this is not gonna work.